### DISCUSSION TITLE
<title>I unlocked 1 badge, when can I find the res

### DISCUSSION THREAD

**Discussion by michael34df** [community/community#64354] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Select Topic Area
Question

Body
So, I unlocked 1 badge on my profile, does anyone know where I can look to find information about getting the rest?

**Comment by santimars** [comment]
Hi, I'm pretty sure you can see information on the other badges at this repo:
https://github.com/Schweinepriester/github-profile-achievements

Its very organized and has good info for what ur looking for. #1

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
